src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/LogisticRegression.cs

```diff
@@ -477,8 +477,7 @@ internal static CommonOutputs.BinaryClassificationOutput TrainBinary(IHostEnviro
     /// <summary>
     /// Computes the standard deviation matrix of each of the non-zero training weights, needed to calculate further the standard deviation,
     /// p-value and z-Score.
-    /// If you need fast calculations, use the <see cref="ComputeLogisticRegressionStandardDeviation"/> implementation in the Microsoft.ML.Mkl.Components package,
-    /// which makes use of hardware acceleration.
+    /// If you need fast calculations, use this class' implementation in the Microsoft.ML.Mkl.Components package which uses Intel Math Kernel Library.
```

Comment:
>If you need fast calculations [](start = 8, length = 29)

Looked at the code, and the MKL implementation is actually the only implementation, so we probably don't need to say "if you need fast calculations" (who would prefer slow calculations anyway? :-)).

Reply:
lol

Reply:
So change to something like "To compute the standard deviation, use this..."

---
In reply to: [278327642](https://github.com/dotnet/machinelearning/pull/3572#discussion_r278327642) [](ancestors = 278327642)

Reply:
Changed to Use this class' implementation in the Microsoft.ML.Mkl.Components package which uses Intel Math Kernel Library....also the hardware acceleration part was a little dramatic